src/main/java/com/pomelers/domain/model/SignUpForm.java

```diff
@@ -0,0 +1,46 @@
+package com.pomelers.domain.model;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import com.pomelers.domain.model.validator.ConfirmPassword;
+import com.pomelers.domain.model.validator.UniqueEmail;
+
+@ConfirmPassword
+public class SignUpForm {

```

Comment:
All validations will be executed at random order.
For example, if the email field is empty, uniqueness check shouldn't be executed.